lib/client/api.go

```diff
@@ -69,7 +69,8 @@ import (
 
 const (
 	// ProfileDir is a directory location where tsh profiles (and session keys) are stored
-	ProfileDir = ".tsh"
+	ProfileDir            = ".tsh"
+	DefaultAddKeysToAgent = "auto"
```

Comment:
define consts for all other possible values and use them in the code
using literal strings throughout the code can lead to subtle bugs due to typos